gui/mozregui/global_prefs.py

```diff
@@ -1,33 +1,34 @@
-from PyQt4.QtCore import QSettings, QVariant
 from PyQt4.QtGui import QDialog
 
 from mozregui.ui.global_prefs import Ui_GlobalPrefs
 from mozregui import patch_requests
 
 from mozregression.network import set_http_session
+from mozregression.config import DEFAULT_CONF_FNAME, get_defaults
+from configobj import ConfigObj
 
 
 def get_prefs():
     """
     Return the global prefs as a dict.
     """
-    settings = QSettings()
+    settings = get_defaults(DEFAULT_CONF_FNAME)
     options = {}
     options['persist'] = \
-        str(settings.value("globalPrefs/persist",
-                           QVariant("")).toString()) or None
+        str(settings.get("globalPrefs/persist",
+                         "").toString()) or None
     options['http_timeout'] = \
-        settings.value("globalPrefs/http_timeout",
-                       QVariant(30.0)).toDouble()[0]
+        settings.get("globalPrefs/http_timeout",
+                     30.0).toDouble()[0]
     return options
 
 
 def save_prefs(options):
-    settings = QSettings()
-    settings.setValue("globalPrefs/persist",
-                      QVariant(options['persist'] or ''))
-    settings.setValue("globalPrefs/http_timeout",
-                      QVariant(options['http_timeout']))
+    settings = get_defaults(DEFAULT_CONF_FNAME)
+    settings.update(ConfigObj("globalPrefs/persist",
+                              (options['persist'] or '')))
+    settings.update(ConfigObj("globalPrefs/http_timeout",
+                              (options['http_timeout'])))
```

Comment:
Well save_prefs should write the new values, but here you are just updating a dict that is lost just after.

So, I think this should be more like:

settings = ConfigObj(DEFAULT_CONF_FNAME)

then update the settings, and at the end write to a file. This have to be tested, using the Global preferences dialog.